images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -37,6 +37,22 @@ function Get-RustVersion {
     return "Rust ${rustVersion}"
 }
 
+function Get-BindgenVersion {
+    return bindgen --version
+}
+
+function Get-CbindgenVersion {
+    return cbindgen --version
+}
+
+function Get-CargoauditVersion {
+    return cargo audit --version
+}
+
+function Get-CargooutdatedVersion {
+    return cargo outdated --version
```

Comment:
Perhaps `Get-CargoAuditVersion` and `Get-CargoOutdatedVersion` names will be more readable.

Reply:
Fixed naming